qiskit/tools/file_io.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2017 IBM RESEARCH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+"""Utilities for File Input/Output."""
+
+
+import os
+import datetime
+import copy
+import json
+from qiskit._qiskiterror import QISKitError
+import qiskit
+import numpy
+
+def convert_qobj_to_json(in_item):
```

Comment:
May be we should change name of this since this isn't actually converting a qobj to json but rather _result+_qobj, so we could just call it something like "_to_json()" like is done with python pandas for instance.